projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.ts

```diff
@@ -54,6 +64,10 @@ export class MessagingComponent implements OnInit {
     );
   }
 
+  get allowedTypes(): Array<string> {
+    return this.messagingConfigs?.attachmentRestrictions?.allowedTypes || [];
```

Comment:
Would there be a case where an empty list would be returned and allow types that were never supposed to be allowed?

Reply:
empty list would mean that no files are allowed.

Reply:
Ahh nevermind I read it wrong